### PR TITLE
Make tech sckills unique, and warn the user of duplicate skills. Mute…

### PR DESCRIPTION
**Pull request by jakobes**
… the latex compile output by default

### CODE REVIEW

**Review by alexanfl — APPROVED** [review]
lgtm

**Comment by alexanfl** [comment]
@jakobes Need to fix merge conflict and perhaps update the `version` in `pyproject.toml`?

**Review by alexanfl — DISMISSED** [review]
@jakobes I tried to resolve the merge conflict, can you check if it's OK? This MR is old, but I think the content is good.

File: cvcreator/parser.py [hunk 1]
```diff
@@ -47,6 +48,8 @@ def cv():
     "Comma-separated list of project tags to include. Use ':' for all."))
 @click.option("-u", "--publications", default="", help=(
     "Comma-separated list of publication tags to include. Use ':' for all."))
+@click.option("-v", "--verbose", is_flag=False, help=(
+    "Do not muffle latex compilation output")
```
Comment (alexanfl) [inline cvcreator/parser.py:52]:
```suggestion
@click.option("-v", "--verbose", is_flag=False, help=("Do not muffle latex compilation output")
```

File: cvcreator/parser.py [hunk 2]
```diff
@@ -47,6 +48,7 @@ def cv():
     "Comma-separated list of project tags to include. Use ':' for all."))
 @click.option("-u", "--publications", default="", help=(
     "Comma-separated list of publication tags to include. Use ':' for all."))
+@click.option("-v", "--verbose", is_flag=False, help=("Do not muffle latex compilation output")
```
Comment (alexanfl) [inline cvcreator/parser.py:51]:
```suggestion
@click.option("-v", "--verbose", is_flag=False, help="Do not muffle latex compilation output")
```

**Comment by jakobes** [comment]
I did a merge with main, now the pipeline is passing. 
Now it's your turn to approve @alexanfl

**Review by alexanfl — APPROVED** [review]
lgtm!

**Pull request merged**